tfe/resource_tfe_organization_token.go

```diff
@@ -68,7 +74,26 @@ func resourceTFEOrganizationTokenCreate(d *schema.ResourceData, meta interface{}
 		log.Printf("[DEBUG] Regenerating existing token for organization: %s", organization)
 	}
 
-	token, err := config.Client.OrganizationTokens.Create(ctx, organization)
+	// Get the token create options.
+	options := tfe.OrganizationTokenCreateOptions{}
+
+	// Check whether the optional expiry was provided.
+	_, expiredAtProvided := d.GetOk("expired_at")
```

Comment:
Small suggestion: I believe that the return value which is getting ignored here (the `_`) is actually going to be the expiredAt value if it exists. So we could change this to

```
expiredAt, expiredAtProvided := d.GetOk("expired_at")
```
and no longer need [line 85](https://github.com/hashicorp/terraform-provider-tfe/pull/844/files#diff-3ba6ff1c80cd647a2c785e0c5f5fb72a8aa985181fb2689b41686131d98bfb0fR85)

Reply:
Great suggestion! I've updated this for both the org and team token.